### ISSUE TITLE
Make a page for showing repositories to contribute to! 

### ISSUE THREAD

**Issue by mananchawla2005** [mananchawla2005/Hacktoberfest-Repo-Checker#51]
Make another separate page using ejs... and js on the backend... which shows different repositories a user can contribute to... 
You can ask the basics steps needed to implement the same from me if at all required... be consistent with the octocat library used for github api calls...

**Comment by ashishnpise** [comment]
Do you want the list to be personally added by a user per pr or do you want to store the searched and verified hacktoberfest verified repos ?

**Comment by Gismo1337** [comment]
may you can use https://github.com/Gismo1337/hacktoberfest-project-list --> https://dev.to/gismo1337/hacktoberfest-projectlist-showcase-3d5f

**Comment by mananchawla2005** [comment]
@AshishisLiquid basically just randomly show the ones with hacktoberfest topic for now... later will create issues for further improvements but rn thats what's required

**Comment by ashishnpise** [comment]
> @AshishisLiquid basically just randomly show the ones with hacktoberfest topic for now... later will create issues for further improvements but rn thats what's required

So we have to show the list hardcoded right as per names?

**Comment by mananchawla2005** [comment]
@AshishisLiquid no you have to fetch it from the api! first and then arrange it in the ejs template

**Comment by mananchawla2005** [comment]
@AshishisLiquid you still working on this?

**Comment by ashishnpise** [comment]
I am looking for api which will provide me lists of repos which have hacktoberfest tag

**Comment by mananchawla2005** [comment]
read octokit docs perhaps @AshishisLiquid

**Comment by dyarawilliams** [comment]
Is this taken... I would like to try this.

**Comment by mananchawla2005** [comment]
@dyarawilliams go ahead!! its not!

**Comment by dyarawilliams** [comment]
@mananchawla2005 Okay Thank you I'll get right to it.

**Comment by mananchawla2005** [comment]
@dyarawilliams any updates?

**Comment by dyarawilliams** [comment]
@mananchawla2005 yes currently working on it right now ... I have a route called repos. I'm working on the front end then I'll utilize the GitHub API

**Comment by dyarawilliams** [comment]
@mananchawla2005 what would you like for each card of the repo to show. I was thinking title of repo, stars, user, and language list

**Comment by mananchawla2005** [comment]
thats perfect! @dyarawilliams

**Comment by dyarawilliams** [comment]
Something like this @mananchawla2005 

![validchecker](https://user-images.githubusercontent.com/15236104/195184518-ba79be3b-98e9-4962-abbb-e3e9d891b75b.PNG)

**Comment by mananchawla2005** [comment]
@dyarawilliams yes thats awesome

**Comment by dyarawilliams** [comment]
- [x] Showing repos to contribute to

![Hacktoberfest-Repository-Checker](https://user-images.githubusercontent.com/15236104/195263359-6bb7eb98-4933-4445-ba8d-4b0eaa1c8a2a.gif)

**Comment by mananchawla2005** [comment]
hey #30 @dyarawilliams would you like to take a look at that as well?

**Comment by dyarawilliams** [comment]
@mananchawla2005 I'll take a look at it
